mithril-common/src/crypto_helper/types/protocol_key.rs

```diff
@@ -50,7 +50,7 @@ where
 
     /// Create an instance from a JSON hex representation
     pub fn from_json_hex(hex_string: &str) -> StdResult<Self> {
-        let key = key_decode_hex::<T>(hex_string).with_context(|| {
+        let key = key_decode_hex::<T>(hex_string.trim()).with_context(|| {
```

Comment:
Is this a problem with this method of with `key_decode_hex` ?

Reply:
You're right, I fixed the problem in the 'key_decode_hex' function instead :+1: